internal/controller/components/kserve/kserve_support.go

```diff
@@ -35,7 +35,11 @@ const DefaultCertificateSecretName = "knative-serving-cert"
 
 var (
 	imageParamMap = map[string]string{
-		"oauth-proxy": "RELATED_IMAGE_OSE_OAUTH_PROXY_IMAGE",
+		"kserve-agent":               "RELATED_IMAGE_KSERVE_AGENT_IMAGE",
```

Comment:
actually, can you please update the prefix  to:
```
RELATED_IMAGE_ODH_
```

Reply:
@riprasad @zdtsw can you please verify if the env names are now correct?

Reply:
@spolti The environment variables are correct.
Please verify that the keys are accurate and approve if everything looks good.

Reply:
just did, looks ok.